{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "413931f1_b8865114",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 4
      },
      "lineNbr": 0,
      "author": {
        "id": 1189020
      },
      "writtenOn": "2020-11-09T13:30:13Z",
      "side": 1,
      "message": "FYI tests were failing due to the crossover with Changhao\u0027s CL.",
      "revId": "1a921b0ac5826a2fc3e8e524b7abc73c1c0e7b28",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}